automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Collapse.scala

```diff
@@ -15,7 +15,7 @@ case class Collapse[A <: FireCloudView](queryString: QueryString, private val in
   }
 
   def toggle(): Unit = {
-    awaitVisible()
+    await visible toggleComponent
```

Comment:
in collapsed state, `Collapse` object is not visible. `awaitVisible` always fails when toggle from collapsed to expanded. changing to check for toggleComponent visibility.